cfgov/jinja2/v1/_includes/macros/time.html

```diff
@@ -41,3 +41,20 @@
     {% endif %}
 </span>
 {% endmacro %}
+
+{% macro render_date_no_markup(datetime,
+                show_value_for={'date':true, 'time':true, 'timezone':true}) %}
+    {# TODO: Replace this with a custom Jinja method that uses
+             https://dateutil.readthedocs.io/ #}
+    {# Displays the date in the format: MMM. DD, YYYY #}
+    {% set format_month = '{dt:%b}'.format(dt = datetime) %}
+    {% if format_month == 'May' %}
+        {# May doesn't have a period. #}
+        {{ '{dt:%b} {dt.day}, {dt.year}'.format(dt = datetime) }}
+    {% elif format_month == 'Sep' %}
+        {# September is four letters. #}
+        {{ 'Sept. {dt.day}, {dt.year}'.format(dt = datetime) }}
+    {% else %}
+        {{ '{dt:%b}. {dt.day}, {dt.year}'.format(dt = datetime) }}
+    {% endif %}
```

Comment:
Thinking about it further, a custom wrapper around strftime that either introduces our own format code or handles `%b` differently from the standard library would mean we could actually use date formatting here (instead of dt.month, dt.day, dt.year) and in all the other places in this changeset like we should be doing.